_includes/scenarios/integrate/succeeding-attributes-to-update-their-values.md

```diff
@@ -0,0 +1,84 @@
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/8e81cd6b-240f-4e41-ae08-9eea13b32974" id="zt0-EawwJWIx"></iframe></div>
+
+### Notifying a peer about a RepositoryAttribute succession
+
+After succeeding a RepositoryAttribute, you can check out [with which of your peers you have previously shared the succeeded Attribute]({% link _docs_use-cases/use-case-consumption-get-shared-versions-of-a-repositoryattribute.md %}) to choose those, you'd like to [notify about the succession]({% link _docs_use-cases/use-case-consumption-notify-peer-about-repositoryattribute-succession.md %}).
+Your own shared IdentityAttributes associated with the peers chosen will be succeeded in the same manner as your RepositoryAttribute before.
+Then, a [Notification]({% link _docs_integrate/data-model-overview.md %}#notification) is sent to the peers, containing a [PeerSharedAttributeSucceededNotificationItem]({% link _docs_integrate/data-model-overview.md %}#peersharedattributesucceedednotificationitem).
+In its `successorContent` property the updated IdentityAttribute is transmitted and automatically a likewise succession at their side will be triggered, such that their LocalAttribute versions replicate the succession chain at your side.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/079d7602-95af-4ddf-8fd5-cca5294038d9" id="Fv0-x9lenepV"></iframe></div>
+
+## Succeeding own RelationshipAttributes
+
+[RelationshipAttributes]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) are, as the name suggests, always associated with a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship).
+Thus, it is not possible to have unshared instances of them.
+Instead, you and your peer will always each have a LocalAttribute with the same RelationshipAttribute as `content`.
+We refer to the LocalAttribute of the `owner` as **own shared RelationshipAttribute** and to the peer's LocalAttribute as **peer shared RelationshipAttribute**.
+
+### Creating and sharing a RelationshipAttribute
+
+Wanting to [create a RelationshipAttribute]({% link _docs_use-cases/use-case-consumption-create-and-share-a-relationshipattribute.md %}) always involves sharing it directly with a peer.
+To this end, a Request containing a [CreateAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#createattributerequestitem) with the RelationshipAttribute will be created and will be sent to the peer.
+Only if the peer accepts this Request, a LocalAttribute with the RelationshipAttribute as `content` will be created at their side and, by receiving the Response, at your side, too.
```

Comment:
"... accepts this Request will a LocalAttribute ... be created at their side ... and **after** receiving ..."